docker-compose.adminer.yaml

```diff
@@ -2,12 +2,13 @@
 services:
   adminer:
     container_name: ddev-${DDEV_SITENAME}-adminer
-    build:
-      context: './adminer'
+    image: shyim/adminerevo:latest
     environment:
       # Use the line below to change the adminer theme.
       # - ADMINER_DESIGN=nette
       - ADMINER_DEFAULT_SERVER=db
+      - ADMINER_DEFAULT_USER=db
+      - ADMINER_DEFAULT_PASSWORD=db
```

Comment:
can we set a predefined DB here?

Reply:
Yes, it is possible via `ADMINER_DEFAULT_DB` (documentation for env variables: https://github.com/shyim/adminerevo-docker#setting-default-login-credentials).

I have added it to this file via commit: [`1daafc6` (#30)](https://github.com/ddev/ddev-adminer/pull/30/commits/1daafc61c955a05667f50503a2377d93db46df45)